03 - CSS Variables/index.html

```diff
@@ -42,9 +48,30 @@ <h2>Update CSS Variables with <span class='hl'>JS</span></h2>
     input {
       width: 100px;
     }
+
+    img {
+      padding: var(--spacing);
+      filter: blur(var(--blur));
+      background-color: var(--base);
+    }
+
+    h2 .hl {
+      color: var(--base);
+    }
   </style>
 
   <script>
+    let inputs = document.querySelectorAll( ".controls input" );
```

Comment:
It's good practice to convert query selections like this (which returns a `NodeList`) to arrays, especially if you're planning to perform a series of array operations. In this case it's just a simple `forEach` loop which is fine.

Reply:
That's great to know, thanks.